### PR TITLE
[STUDIO-1547] Azure DevOps extension: Fixing Node 6 execution handler warning message

### PR DESCRIPTION
**Pull request by Minh-Chien**
## Code Review Checklist

- [x] Check base branch.
- [x] Write down ticket description on JIRA.
- [x] Verify all code changes are formatted.
- [ ] Review SonarCloud code quality report.

### CODE REVIEW

File: katalonTask/package.json [hunk 1]
```diff
@@ -52,6 +52,7 @@
     "eslint": "^5.16.0",
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.16.0",
-    "pkg": "^4.2.4"
+    "pkg": "^4.2.4",
+    "typescript": "^4.0.2"
```
Comment (huyennguyen-katalon) [inline katalonTask/package.json:56]: Why did you add this??? Do we use it

Comment (Minh-Chien) [reply katalonTask/package.json]: I removed the unnecessary package. Please help to review the ticket.

**Comment by duyluonganh** [comment]
@Minh-Chien @huyennguyen-katalon 
Please also update the plugin version in:
https://github.com/katalon-studio/katalon-azure-devops-extension/blob/master/katalonTask/task.json
https://github.com/katalon-studio/katalon-azure-devops-extension/blob/master/katalonTask/package.json\

Please refer to my previous commit: https://github.com/katalon-studio/katalon-azure-devops-extension/commit/be48f5d292e878f263bdcc70db0124d1e2a6b92a

**Comment by huyennguyen-katalon** [comment]
Yep, I've just updated.

**Review by duyluonganh — APPROVED** [review]
